### ISSUE TITLE
Is Mapbox planning to upgrade(this project) to ARFoundation?

### ISSUE THREAD

**Issue by saszer** [mapbox/mapbox-unity-sdk#1747]
From my current understanding, 

This project is Using ARInterface Library with custom imported GoogleARCore and ARkit. 

1, This creates conflicts if any project is using newer inbuilt Unity Packages for ARCore and Kit. 
2. ARInterface is superseded by ARFoundation(in integration with two packages above), however, it's not as simple as replacing Interface for Foundation in this project as scripts for AR alignment has dependencies on ARInterface and starts to throw a number of errors. 

It'll be great if the whole example project is upgraded to ARFoundation, which will now be getting continued support from Unity.

**Comment by kintovt** [comment]
@saszer did you get example projects to work?

**Comment by saszer** [comment]
@kintovt yes, although I dont recall the exact steps as this was a while ago between a lot of experimentation. I do have ARFoundation in my packages and GoogleARCore in the project folder(which comes with mapbox). 

No need to get arcore or arkit from packages, just import ARFoundation in packages. I might have commented out some of the code which was throwing errors.

**Comment by Markovicho** [comment]
@saszer If you are planning to submit your application to the public it's pretty risky to use those deprecated plugins shipped with mapbox. It may be working for the moment (which i am not sure since the release of iOS 14.4) but it's only a matter of time ARCore/ARKit will show up with the first API change breaking the compatibility. Those librarys are updated automatically with the mobile OS over playstore in the case of Android and bound with the iOS Version on apples side. 

Refer to this issue for an official answer regarding your title question : 
https://github.com/mapbox/mapbox-unity-sdk/issues/1619

**Comment by brnkhy** [comment]
I'm afraid Ar support in the SDK is really outdated and there most likely won't be a fix/update any time soon. :(
